RNTester/js/PanResponderExample.js

```diff
@@ -10,29 +10,27 @@
 
 'use strict';
 
-var React = require('react');
-var createReactClass = require('create-react-class');
-var ReactNative = require('react-native');
-var {PanResponder, StyleSheet, View} = ReactNative;
+const React = require('react');
+const ReactNative = require('react-native');
+const {PanResponder, StyleSheet, View} = ReactNative;
 
-var CIRCLE_SIZE = 80;
+const CIRCLE_SIZE = 80;
 
-var PanResponderExample = createReactClass({
-  displayName: 'PanResponderExample',
+class PanResponderExample extends React.Component<{}> {
+  static displayName: ?string = 'PanResponderExample';
 
   statics: {
     title: 'PanResponder Sample',
-    description:
-      'Shows the use of PanResponder to provide basic gesture handling.',
-  },
+    description: 'Shows the use of PanResponder to provide basic gesture handling.',
+  };
 
-  _panResponder: {},
-  _previousLeft: 0,
-  _previousTop: 0,
-  _circleStyles: {},
-  circle: (null: ?{setNativeProps(props: Object): void}),
+  _panResponder: Object = {};
```

Comment:
PanResponder has Flow typings. I think we should use them in here.

Reply:
I use PanResponderInstance... Is this correct....?